### PR TITLE
Update versionadded/versionchanged in master according to the latest releases

### LINKED ISSUE #2133: Update versionadded/versionchanged in master accordning to the latest releases
Currently, we are manually updating versionadded/versionchanged for each v0.4.X release:
- https://github.com/pytorch/ignite/commit/eee8dc5159eaec741d677c8a6db9f7ba00d50afa
- https://github.com/pytorch/ignite/commit/84e51c3b4987c16f6dd27a6abd2f9ff95da3dc6c
- https://github.com/pytorch/ignite/commit/84e51c3b4987c16f6dd27a6abd2f9ff95da3dc6c

Tasks to do to fix this issue:
- [ ] Let's update versionadded/versionchanged in the master according to v0.4.6 release: https://github.com/pytorch/ignite/tree/0.4.6 . We need to find all mentions of `0.4.5` and `0.4.6` in v0.4.6 and port them to the master.
- [ ] Add `.. versionadded:: 0.4.6` as [here](https://github.com/pytorch/ignite/blob/84e51c3b4987c16f6dd27a6abd2f9ff95da3dc6c/ignite/handlers/stores.py#L37) to `FastaiLRFinder` here: https://github.com/pytorch/ignite/blob/3ef85e1b7af58cfbcf2033d62085a759aa722cc1/ignite/handlers/lr_finder.py#L73

### PR DESCRIPTION
**Pull request by Chandan-h-509**
Fixes #2133 

Description: 
Updated versionadded/versionchanged in the master according to v0.4.6 release.
Add .. versionadded:: 0.4.6 to FastaiLRFinder.

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [x] Documentation is updated (if required)

### CODE REVIEW

File: ignite/__init__.py [hunk 1]
```diff
@@ -6,4 +6,4 @@
 import ignite.metrics
 import ignite.utils
 
-__version__ = "0.5.0"
+__version__ = "0.4.6"
```
Comment (vfdev-5) [inline ignite/__init__.py:9]: Let's revert this change, as it is master's version

Comment (Chandan-h-509) [reply ignite/__init__.py]: Made the change. Please review

**Review by trsvchn — CHANGES_REQUESTED** [review]
Thank you for PR, @Chandan-h-509 !

I think we are still missing one file here

https://github.com/pytorch/ignite/blob/84e51c3b4987c16f6dd27a6abd2f9ff95da3dc6c/ignite/contrib/metrics/regression/fractional_bias.py#L37

**Comment by Chandan-h-509** [comment]
Thank you @vfdev-5 .
I have updated the necessary change.
Please review.

**Review by trsvchn — APPROVED** [review]
Thanks @Chandan-h-509! LGTM!

**Comment by Chandan-h-509** [comment]
Thanks @trsvchn for the opportunity...Pleasure contributing.

**Comment by trsvchn** [comment]
@Chandan-h-509 Thank you for your contribution! Looking forward to seeing more PR from you! ;) Have a great day!
